### DISCUSSION TITLE
Hardware

### DISCUSSION THREAD

**Discussion by f2daz** [ehorvat1/NES-MEP-Reader#4] [General]
Hi @ehorvat1,

du hast dir ja bestimmt mehrere PCBs drucken lassen. Hättest du eventuell noch eines über? Würde das gerne bei mir zuhause auch testen. Bin hier in Deutschland bei einem der wenigen Energieversorger, der auch Echelon Zähler nutzt ;)

Vielen Dank schon mal und schöne Grüße
Felix

**Comment by ehorvat1** [comment]
Servus Felix,
ja ich hab noch ein paar PCBs....sogar noch eine die schon bestückt ist. Kann ich dir senden...schreib mir dazu doch direkt auf engelbert.horvat@gmx.net mal an. LG, Bert
